src/patterns/help-users-contact-a-department/all-contact-information/index.njk

```diff
@@ -0,0 +1,26 @@
+---
+title: All contact information
+layout: layout-example.njk
+---
+
+<p class="govuk-body govuk-!-font-weight-bold">Get help with your application</p>
+
+<ul class="govuk-list">
+  <li>Telephone: 020 7946 0101</li>
+  <li>Textphone: 020 7946 0102</li>
+  <li><span class="govuk-visually-hidden">Opening times: </span>Monday to Friday, 8am to 6pm</li>
```

Comment:
Where possible we try to keep content visible to all users, could you explain your thoughts on hiding some content here?

Reply:
Changes made as per suggestion.